llvm/include/llvm/ADT/STLFunctionalExtras.h

```diff
@@ -69,6 +69,10 @@ class function_ref<Ret(Params...)> {
   }
 
   explicit operator bool() const { return callback; }
+
+  bool operator==(const function_ref<Ret(Params...)> &Other) const {
```

Comment:
Why is this needed now?

Reply:
This is needed now because of its use in `cl::OptionValueCopy<FnT>`, which uses the `==` operator.